packages/camera/camera_avfoundation/ios/Classes/FLTCam.m

```diff
@@ -399,6 +403,97 @@ - (BOOL)setCaptureSessionPreset:(FLTResolutionPreset)resolutionPreset withError:
         return NO;
       }
   }
+  return YES;
+}
+
+- (BOOL)setCaptureSessionForPhoto:(FLTResolutionPreset)resolutionPreset
+                        withError:(NSError **)error {
+  switch (resolutionPreset) {
+    case FLTResolutionPresetMax:
+    case FLTResolutionPresetUltraHigh:
+      if ([_videoCaptureSession canSetSessionPreset:AVCaptureSessionPresetPhoto]) {
```

Comment:
does `AVCaptureSessionPresetPhoto` give the max resolution? it's hard to tell from its name. maybe add a comment if that's the case.